src/components/CountyMap/CountyMap.js

```diff
@@ -12,24 +12,17 @@ import { FIPS_CODE_TO_CALCULATED_INTERVENTION_COLOR } from 'enums/interventions'
 import { COLOR_MAP } from 'enums/interventions';
 import { CountyMapWrapper, CountyMapLayerWrapper } from './CountyMap.style';
 
-const CountyMap = ({
-  selectedCounty,
-  setSelectedCounty,
-  fill,
-  stateSummary = {},
-}) => {
+const CountyMap = ({ selectedCounty, setSelectedCounty }) => {
   let { stateId } = useParams();
   stateId = stateId.toUpperCase();
   const state = STATE_CENTERS[stateId];
   const counties = require(`./countyTopoJson/${stateId}.json`);
-  const countiesWithData =
-    stateSummary && stateSummary.counties_with_data
-      ? stateSummary.counties_with_data
-      : [];
   const [content, setContent] = useState('');
 
   const getFillColor = geoId => {
-    return FIPS_CODE_TO_CALCULATED_INTERVENTION_COLOR[geoId] || fill;
```

Comment:
`fill` was actually never getting used because this function only gets called for the counties in the summary file and (because of how calculated-interventions.ts works) every county in the summary file is guaranteed an entry in the FIPS_CODE_TO_CALCULATED_INTERVENTION_COLOR map.

Also the passed in `fill` color was "wrong" (it was the state's intervention color), so I have removed the `fill` prop entirely.